OpenSim/Simulation/Model/PathPoint.h

```diff
@@ -35,88 +35,76 @@ class PhysicalFrame;
 //=============================================================================
 //=============================================================================
 /**
- * A class implementing a path point.
- *
- * @author Peter Loan
- * @version 1.0
+ * A PathPoint that is stationary with respect to parent's PhysicalFrame
```

Comment:
Should also mention that PathPoint uses a Station under the hood so that the description of the "location" property makes sense.